queries/fsh/highlights.scm

```diff
@@ -29,7 +29,7 @@
   "Logical"
   "Resource"
   "RuleSet"
-] @keyword
+] @type.definition
```

Comment:
`@type.definition` is only used for type aliases.

Reply:
Ah okay, maybe type.builtin makes more sense for this one

Reply:
After a quick look at the docs, keeping it as `@keyword` is probably the best option.

Reply:
Got it